bot/prompt.py

```diff
@@ -209,14 +217,15 @@ def gen_abortion_prompt():
         (5.2, gen_abortion_prompt_I(accused)),
         (2.6, gen_abortion_prompt_My(accused))
     ]
-    return random_select_weighted_list(abortion_prompts)
+    return add_typos(random_select_weighted_list(abortion_prompts))
 
 if __name__ == "__main__":
     total_number = 2000000
     sample_abortion_prompts = { gen_abortion_prompt() for k in range(total_number) }
+    print('succ')
```

Comment:
Lol, we probably don't need that print statement.

Reply:
I was doing a bit of print debugging. It has been removed.